e2e/_suites/fleet/world.go

```diff
@@ -13,16 +13,15 @@ import (
 
 // IngestManagerTestSuite represents a test suite, holding references to the pieces needed to run the tests
 type IngestManagerTestSuite struct {
-	Fleet      *FleetTestSuite
-	StandAlone *StandAloneTestSuite
+	Fleet *FleetTestSuite
```

Comment:
For follow-up, we can simply merge everything into the `IngestManagerTestSuite`, but I'm totally OK doing that in a follow-up PR.